nxcomp/Log.h

```diff
@@ -148,7 +148,7 @@ class NXLog
 	} per_thread_data;
 
 
-	per_thread_data* get_data()
+	per_thread_data* get_data_int() const
```

Comment:
Hmm... I'm not sure that const is right here. This function might change/allocate "opaque" data used by everything here... but I really have no idea whether this qualifies as non-const or not.